src/test/rpc/LedgerEntry_test.cpp

```diff
@@ -31,40 +31,435 @@
 #include <xrpl/protocol/STXChainBridge.h>
 #include <xrpl/protocol/jss.h>
 
+#if (defined(__clang_major__) && __clang_major__ < 15)
+#include <experimental/source_location>
+using source_location = std::experimental::source_location;
+#else
+#include <source_location>
+using std::source_location;
+#endif
 namespace ripple {
 
 namespace test {
 
 class LedgerEntry_test : public beast::unit_test::suite
 {
+    enum FieldType {
+        AccountField,
+        StringField,
+        HashField,
+        CurrencyField,
+        ArrayField,
+        ObjectField,
+        HashOrObjectField,
+        UInt32Field,
+        UInt64Field,
+        BlobField,
+        TwoAccountArrayField,
+    };
+
+    std::vector<std::pair<Json::StaticString, FieldType>> mappings{
+        {jss::account, AccountField},
+        {jss::accounts, TwoAccountArrayField},
+        {jss::authorize, AccountField},
+        {jss::authorized, AccountField},
+        {jss::currency, CurrencyField},
+        {jss::credential_type, BlobField},
```

Comment:
We should probable sort this list as well

Reply:
Fixed in [f380124](https://github.com/XRPLF/rippled/pull/5237/commits/f380124966119c33193e8b91fdb857d521f70953)